File: skeleton/src/main/java/me/shkschneider/skeleton/ui/transforms/DefaultTransformer.java
package me.shkschneider.skeleton.ui.transforms;

import android.view.View;

// <https://github.com/ToxicBakery/ViewPagerTransforms>
public class DefaultTransformer extends ABaseTransformer {

	@Override
	protected void onTransform(final View view, final float position) {
        // Empty
	}

	@Override
	public boolean isPagingEnabled() {
		return true;
	}

}

File: library/src/main/kotlin/me/shkschneider/skeleton/ui/transforms/DefaultTransformer.kt
package me.shkschneider.skeleton.ui.transforms

import android.view.View

// <https://github.com/ToxicBakery/ViewPagerTransforms>
class DefaultTransformer : ABaseTransformer() {

    override fun isPagingEnabled(): Boolean {
        return true
    }

    override fun onTransform(page: View, position: Float) {
        // Empty
    }

}
